molo/core/migrations/0004_configure_root_page.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def configure_root_page(apps, schema_editor):
+    # Get models
+    ContentType = apps.get_model('contenttypes.ContentType')
+    Site = apps.get_model('wagtailcore.Site')
+    Main = apps.get_model('core.Main')
+    HomePage = apps.get_model('core.HomePage')
+
+    # Delete the default homepage
+    HomePage.objects.all().delete()
+
+    # Create content type for main model
+    main_content_type, created = ContentType.objects.get_or_create(
+        model='main', app_label='core', defaults={'name': 'Main'})
+
+    # Create a new homepage
+    main = Main.objects.create(
+        title="Main",
+        slug='main',
+        content_type=main_content_type,
+        path='00010001',
```

Comment:
it's the tree syntax they use. It denotes the 1st child of the root element. e.g `0001(root)0001(1st child)`. You can also start doing things like `00010002`. It's generally handled by wagtail code except when you're creating models yourself.